asm-iap/cluster/iap-operator.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This overlay adds compatibility for upgrades to clusters using GCLB
+# for certain versions of istioctl
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    ingressGateways:
+    - enabled: true
+      name: istio-ingressgateway
+      k8s:
+        service:
+          type: NodePort
+          ports:
+            - name: status-port
+              nodePort: 31223
+              port: 15021
+              protocol: TCP
+              targetPort: 15021
+            - port: 80
+              targetPort: 8080
+              name: http2
+            - port: 443
+              targetPort: 8443
+              name: https
+            - port: 15012
+              targetPort: 15012
+              name: tcp-istiod
+            - port: 15443
+              targetPort: 15443
+              name: tls
```

Comment:
insert into istio-operator.yaml:26 should do the job.

Reply:
Makes sense